package.json

```diff
@@ -120,6 +120,7 @@
     "google-auth-library": "^7.11.0",
     "inquirer": "^8.2.0",
     "js-yaml": "^3.13.1",
+    "JSONStream": "^1.3.5",
```

Comment:
We should not add this import. It is a small, deprecated utility that hasn't been maintained for 4 years, (https://github.com/dominictarr/JSONStream/blob/master/index.js#L17). Use the already imported json-stream instead.

Take a look at https://github.com/firebase/firebase-tools/pull/4027 for an example of how to replace JSONstream with stream-json

Reply:
Done.